insights/parsers/kdump.py

```diff
@@ -205,19 +218,39 @@ def using_local_disk(self):
         """
         Is kdump configured to only use local disk?
 
-        The logic used here is the first of these conditions:
+        Several target types:
 
-        * If 'raw' is given as an option, then the dump is local.
+        * If 'raw' is given, then the dump is local.
         * If 'ssh', 'net', 'nfs', or 'nfs4' is given, then the dump is NOT local.
+        * If '<fs type> <partition>' is given, then the dump is local.
         * Otherwise, the dump is local.
+
+        Since only one target could be set, the logic used here is checking
+        if remote target is used, return True for not.
         """
-        # The previous version used iteration across self.data.keys(), which
-        # is of course non-repeatable because hash key ordering may change.
-        # So we reverted to logic.
-        return ('raw' in self.data) or (not (
-            'ssh' in self.data or 'net' in self.data or
-            'nfs' in self.data or 'nfs4' in self.data)
-        )
+        return not ('ssh' in self.data or 'net' in self.data or
+                    'nfs' in self.data or 'nfs4' in self.data)
+
+    def _check_target_conflict(self):
```

Comment:
As an effort, the `self.target` is got from this method. Better return it as a return value directly.

Btw, how do you think to rename it to `_parse_target`?, e.g.:
```
self.target = self._parse_target
```